src/components/Explore/utils.ts

```diff
@@ -118,6 +196,8 @@ export const exploreMetricData: {
     title: 'Deaths',
     name: 'Deaths',
     chartId: 'deaths',
+    dataMeasure: DataMeasure.INTEGER,
+    yAxisDecimalPlaces: 1,
```

Comment:
`yAxisDecimalPlaces` makes sure deaths/icu hospitalizations charts show the precision of 1 decimal place, since the numbers are getting small and the y axis in some cases was reading 0, 0, 0, 1, 1, etc.